### PR TITLE
[DT-470][risk=no] Adding tanagraEnabled flag to CdrVersion

### PR DESCRIPTION
**Pull request by freemabd**
This PR adds the tanagraEnabled flag to the CdrVersion model. 

**NOTE:** This flag will replace the feature flag `enableTanagra`, but we don't want to change references to this before our demo next Thursday.

This should wrap any tanagra UI and API code that we don't want exposed while in development.

UI example: 
`if (cdrVersion.tanagraEnabled) { }`

API example:
`if (cdrVersion.getTanagraEnabled()) { }`

---
**PR checklist**

- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

**Review by dolbeew — APPROVED** [review]
Deployed locally and confirmed the `tanagraEnabled` property on the `cdrVersionsByTier` response

File: api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java [hunk 1]
```diff
@@ -192,6 +192,8 @@ public ResponseEntity<Workspace> createWorkspace(Workspace workspace) throws Bad
     }
     final Workspace createdWorkspace = workspaceMapper.toApiWorkspace(dbWorkspace, fcWorkspace);
     workspaceAuditor.fireCreateAction(createdWorkspace, dbWorkspace.getWorkspaceId());
+    if (cdrVersion.getTanagraEnabled()) {}
+
```
Comment (chenchals) [inline api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java:196]: Is this just a place holder? Maybe you should mark this-location in the ticket that will implement the logic here and remove these lines. 195-196 code does nothing

Comment (freemabd) [reply api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java]: Thanks, was testing if the flag was accessible and forgot to remove this line of code. Removed.

File: api/src/test/java/org/pmiops/workbench/cdr/CdrVersionServiceTest.java [hunk 2]
```diff
@@ -304,7 +313,7 @@ private void testGetCdrVersionsHasDataType(Predicate<CdrVersion> hasType) {
     assertThat(cdrVersions.stream().anyMatch(hasType)).isFalse();
 
     makeCdrVersion(
-        5L, true, "Test CDR With Data Types", registeredTier, "wgs", true, true, true, true);
+        5L, true, "Test CDR With Data Types", registeredTier, "wgs", true, true, true, true, true);
```
Comment (chenchals) [inline api/src/test/java/org/pmiops/workbench/cdr/CdrVersionServiceTest.java:316]: so, makeCdrVersion-> sets `tanagraEnabled` to true, but I do not see a check to test a call to check if `getTanagraEnabled` from `DbCdrVersion` returns `true`

**Pull request merged**